titan-compiler/symtab.lua

```diff
@@ -13,7 +13,6 @@ end
 function symtab:close_block()
     local last = self.blocks[#self.blocks]
     table.remove(self.blocks)
-    return last
```

Comment:
I thinks it is OK to remove this, but you should also remove the variable declaration at line 14.

Reply:
Of course!